internal/database/webhooks.go

```diff
@@ -140,18 +179,23 @@ func (s *webhookStore) Delete(ctx context.Context, id uuid.UUID) error {
 	}
 
 	if rowsAffected == 0 {
-		return errors.Wrap(&WebhookNotFoundError{ID: id}, "failed to delete webhook")
+		return errors.Wrap(&WebhookNotFoundError{UUID: id}, "failed to delete webhook")
 	}
 	return nil
 }
 
 // WebhookNotFoundError occurs when a webhook is not found.
 type WebhookNotFoundError struct {
-	ID uuid.UUID
+	ID   int32
+	UUID uuid.UUID
 }
 
 func (w *WebhookNotFoundError) Error() string {
-	return fmt.Sprintf("webhook with UUID %s not found", w.ID)
+	if w.UUID != uuid.Nil {
+		return fmt.Sprintf("webhook with UUID %s not found", w.UUID)
+	} else {
+		return fmt.Sprintf("webhook with ID=\"%d\" not found", w.ID)
```

Comment:
It's a number, no quoting needed :)

```suggestion
		return fmt.Sprintf("webhook with ID %d not found", w.ID)
```